src/pages/FuFit.jsx

```diff
@@ -0,0 +1,184 @@
+                        FUSION's athletic programs aim to build a strong community, enhance physical well-being, and support mental 
+                        health through a range of recreational and organized activities. By providing opportunities for exercise, we 
+                        encourage our members to stay active and healthy. Our organized sports activities foster teamwork and camaraderie, 
+                        while our workshops offer a supportive space for relaxation and emotional well-being. Together, these programs 
+                        ensure that our members have the resources they need to thrive both physically and mentally.
+                        </p>
+                    </div>
+                </div>
+            </section>
+
+            {/* Get moving */}
+            <section className="w-full h-[540px] bg-white flex flex-row relative">
+                <div className="absolute inset-0 z-5">
+                    <img src={fufit_bg1} className="w-full h-[540px] object-cover"></img>
+                </div>
+
+                <div className="w-[48%] h-full flex justify-end relative z-10">
+                    <div className="w-[72%] h-full flex justify-end">
+                        <img src={fufit_basketball} className="w-[490px] h-[54%] mt-[20%] "></img>
+                        <img src={fufit_dots1} className="w-20 h-16 absolute transform translate-y-[570%] z-20"/>
+                    </div>
+                </div>
+                
+                <div className="w-[52%] h-full relative z-10"> 
+                    <div className="w-[64%] h-full flex flex-col ml-[14%]">
+                        <div className="w-full h-[76%] bg-[#F4F4F4] mt-10 rounded-t-[60px] rounded-br-[60px]">
+                            <div className="w-[85%] ml-[7%]">
+                                <h1 className="font-[700] text-[#FFA523] mt-[7%] tracking-[.1em]">FUSION FIT &gt;&gt;</h1>
+                                <h1 className="text-black text-3xl font-[700] mt-[3%] ">Get Moving and Feel the Momentum</h1>
+                                <hr className="w-16 h-1  bg-[#FFA523] border-0 rounded mt-[5%]"/>
+                                <p className="text-xs text-black mt-[5%] leading-[22px]">Our recreational exercise program provides general members 
+                                an outlet to stay active and healthy. Participate in a variety of activities designed to suit all fitness levels, 
+                                including basketball, volleyball, soccer, flag football, ultimate frisbee, pickleball, kickball, yoga, group workouts, 
+                                and many more. Whether you are looking to work on your fitness goals or simply want to stay active, FUSION Fits have 
+                                something to offer for everyone at all levels. Join us to stay active, connect with fellow members, and make fitness 
+                                an enjoyable part of your routine!</p>
```

Comment:
The bottom part of this text box is pretty elongated, looking at the figma it seems that the text is taking up the full space. Might want to change the padding stuff and play around with it,